### DISCUSSION TITLE
Annoying arrow symbol

### DISCUSSION THREAD

**Discussion by matt-joecoffee** [starship/starship#3093] [Q&A] [answered]
<img width="75" alt="Screen Shot 2021-09-24 at 3 58 00 PM" src="https://user-images.githubusercontent.com/86847089/134747615-2661473c-3d26-4486-8edb-da917f35fa07.png">

Does anyone know what that blue arrow/caret is in the corner, and how can I make it go away?

I'm in iTerm2 and using bash, if that helps.

**Comment by chipbuster** [comment, marked as answer]
That is the [iTerm2 prompt mark](https://iterm2.com/documentation-shell-integration.html). It's part of iTerm's shell integrations, not starship.

If you installed it via the default method, you should be able to get rid of it by looking for a line in your `.bashrc` that say something like 

```
source ~/.iterm2_shell_integration.bash
```

and deleting it. You may also want to delete the file `~/.iterm2_shell_integration.bash` afterwards.

**Comment by matt-joecoffee** [comment]
Amazing! I found the line in the .bash_profile file. Thank you SO MUCH for pointing me the in the right direction.

**Comment by benkeil** [comment]
I have the opposite problem, how can I make that the marks appear? Everything l looks proper configured...
